evap/static/ts/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const selectOrError = <T extends Element>(elementId: string): T => {
+    const elem = document.querySelector<T>(elementId);
+    if (elem === null) throw new Error(`Element with id ${elementId} not found`);
```

Comment:
I wanted to suggest this change:
```suggestion
    assert(elem !== null, `Element with id ${elementId} not found`);
```
But in the current setup, tsc cannot verify that elem is not null after that, unless we tell it that the assert functions really does assert. For that, we need to make `condition` to type `unknown` and not use arrow functions:
```javascript
export function assert(condition: unknown, message: string = "Assertion Failed"): asserts condition {
    if (!condition) throw new Error(message);
}
```